pkg/volume/fc/fc.go

```diff
@@ -416,8 +416,8 @@ func (b *fcDiskMapper) SetUpDevice() (string, error) {
 	return "", nil
 }
 
-func (b *fcDiskMapper) MapDevice(devicePath, globalMapPath, volumeMapPath, volumeMapName string, podUID types.UID) error {
-	return util.MapBlockVolume(devicePath, globalMapPath, volumeMapPath, volumeMapName, podUID)
+func (b *fcDiskMapper) MapPodDevice() error {
+	return nil
```

Comment:
looks like most (many?) plugins have `nil` implementations for this. Should this be a separate interface that these plugins just don't implement? Something like `BlockVolumeMapper`